IFC4x3/PropertyConstants/u/UNSET_1n8r4XYi950Rj6LjJ0X9h_.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="UNSET_1n8r4XYi950Rj6LjJ0X9h_" Name="UNSET" UniqueId="71235121-8ac2-4501-bb46-56d4c0849afe">
```

Comment:
Is there a need for a new UNSET constant? Why not NOTDEFINED?

Reply:
To address with @aothms